src/compas/rpc/services/special.py

```diff
@@ -0,0 +1,42 @@
+"""
+Minimal example of a "special" service.
```

Comment:
I'm not very convinced about the naming, "special" but why? What's special about it? :) Do you mean "Custom" or "UserDefined"?

Reply:
not it is just an example. perhaps it should be removed and the example added to the docs

Reply:
👍